tests/vision/test_download.py

```diff
@@ -0,0 +1,26 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+from flash.vision.classification.dataset import hymenoptera_data_download
+from flash.vision.embedding.model_map import _load_bolts_model
+
+
+@pytest.mark.parametrize("name", ['simclr-imagenet', 'swav-imagenet'])
+def test_load_bolts(name):
+    _load_bolts_model(name)
+
+
+def test_pull_hymenoptera(tmpdir):
+    hymenoptera_data_download(path=tmpdir)
```

Comment:
Let's download titanic instead which is smaller. This function `hymenoptera_data_download` can be deleted too.